streams/src/main/java/org/apache/kafka/streams/kstream/internals/SerializedInternal.java

```diff
@@ -19,6 +19,7 @@
 import org.apache.kafka.common.serialization.Serde;
 import org.apache.kafka.streams.kstream.Serialized;
 
+@SuppressWarnings("deprecation")
```

Comment:
Ditto to `KStreamImpl`: it might be better to mark this class `@Deprecated`

Reply:
ack